include/roaring/portability.h

```diff
@@ -197,7 +197,7 @@ static inline int roaring_trailing_zeroes(unsigned long long input_num) {
 
 /* wrappers for Visual Studio built-ins that look like gcc built-ins __builtin_clzll */
 /* result might be undefined when input_num is zero */
-inline int roaring_leading_zeroes(unsigned long long input_num) {
+static inline int roaring_leading_zeroes(unsigned long long input_num) {
```

Comment:
As above, is there a particular reason to prefer static here?

Reply:
this is just a replacement for builtin_clz, not exported.